moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDescription.js

```diff
@@ -55,10 +54,10 @@ const ProjectSummaryProjectDescription = ({
         projectId: projectId,
         description: description,
       },
+      refetchQueries: [{ query: listViewQuery }, "ProjectSummary"],
```

Comment:
So—here's the result of all that fussing with conext! you can pass it to `refetchQueries`. any queries specified here get refetched in the background—thereby updating the cache.

As you can see, we are passing the raw `gql` of the `listViewQuery`, but also the name of summary query, `ProjectSummary`. Wouldn't it be nice if we could pass the name `ProjectListView` instead of fussing with contex? I think so too! :/

[Refetching docs](https://www.apollographql.com/docs/react/data/refetching).

Reply:
In any case—I included this as a working example, because we really do want to make sure we're refetching the list view query in the background, so that the list view table does not re-render a few seconds after we navigate back to it. 

this will make more sense when you test it yourself.
1. Load the list view
2. Navigate to any project **except** the project in the first row of the list view
3. Edit the project's name.
4. Navigate back to the project list view.

Observer how there is a brief lag before the list view re-renders and the project you just ended moves to the top.

What's going on:
1. You make a project edit.
2. Hasura event triggers cause the project's `updated_at` date to update
3. You navigate to the project list
4. The project list data is first loaded from the cache, then it is re-hydrated with network data
5. The project list is sorted by `updated_at`, so the project you just edited is re-ordered to the top.

Reply:
Nice! It cool to see `refetchQueries` in action after reading a little about it. 🚀 It is always interesting to face an issue caused by previous logic that was doing just fine on its own! I agree about the DB trigger as a fix.